app/src/pages/inside/launchesPage/historyView/historyTable/historyItemsGrid/historyItem/historyItem.scss

```diff
@@ -0,0 +1,98 @@
+.history-item-wrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  min-width: 62px;
+  min-height: 50px;
+  padding: 4px;
+  font-size: 12px;
+  text-align: center;
+  font-family: $FONT-REGULAR;
+  line-height: 1;
+}
+
+.history-item-status-FAILED {
+  background-color: lighten($COLOR--failed, 10%);
+}
+
+.history-item-status-SKIPPED {
+  background-color: lighten($COLOR--skipped, 10%);
+}
+
+.history-item-status-MANY {
+  background: $COLOR--light-yellow;
+  color: $COLOR--gray-47;
+}
+
+.history-item-status-PASSED {
+  background-color: lighten($COLOR--passed, 10%);
+}
+
+.history-item-status-RESETED {
+  color: $COLOR--gray-80;
+  background: $COLOR--white;
+}
+
+.history-item-status-NOT_FOUND {
+  color: $COLOR--gray-80;
+  background: $COLOR--white;
+}
+
+.launch-status-IN_PROGRESS {
+  background-color: #c5cae9;
```

Comment:
It's not allowed to define colors directly (without using of existing color variables). If you see new color(not defined in palette) it's a reason to have a talk with our UX Designer.